builder/metadata.go

```diff
@@ -172,9 +172,20 @@ Latest upstream in format: %d
 // upstream version is 0, then the latest upstream version in the current
 // upstream format will be taken instead.
 func (b *Builder) UpdateVersions(nextMix, nextUpstream uint32) error {
-	format, _, latest, err := b.getUpstreamFormatRange(b.UpstreamVer)
-	if err != nil {
-		return err
+	var format string
+	var nextFormat string
```

Comment:
can you please put this closer to where it is actually used on line 202? Unless you *like* C89 variable declaration requirements...

Reply:
I mean, this could also be C99 style ;)

Reply:
It has to happen on 200 because it's used out of that scope, moved it though.